packages/app/components/Settings/SettingsPage.tsx

```diff
@@ -0,0 +1,59 @@
+import { BellOutlined, EditOutlined } from "@ant-design/icons";
+import { Col, Menu, Row, Space } from "antd";
+import React, { ReactElement, useState } from "react";
+import { useProfile } from "../../hooks/useProfile";
+import AvatarWithInitals from "../common/AvatarWithInitials";
+import { VerticalDivider } from "../Queue/QueueListSharedComponents";
+import NotificationsSettings from "./NotificationsSettings";
+import ProfileSettings from "./ProfileSettings";
+
+export enum SettingsOptions {
+  PROFILE = "PROFILE",
+  NOTIFICATIONS = "NOTIFICATIONS",
+}
+
+interface SettingsPageProps {
+  defaultPage: SettingsOptions;
+}
+
+export default function SettingsPage({
+  defaultPage,
+}: SettingsPageProps): ReactElement {
+  const profile = useProfile();
+  const [currentSettings, setCurrentSettings] = useState(
+    defaultPage || SettingsOptions.PROFILE
+  );
+
+  return (
+    <Row>
+      <Col span={4} style={{ textAlign: "center" }}>
+        <AvatarWithInitals
+          style={{ marginTop: "60px", marginBottom: "60px" }}
+          name={profile?.name}
+          size={250}
+          fontSize={96}
+        />
+        <Menu onClick={(e) => setCurrentSettings(e.key as SettingsOptions)}>
+          <Menu.Item key={SettingsOptions.PROFILE} icon={<EditOutlined />}>
+            Edit Profile
+          </Menu.Item>
+          <Menu.Item
+            key={SettingsOptions.NOTIFICATIONS}
+            icon={<BellOutlined />}
+          >
+            Notifications Settings
+          </Menu.Item>
+        </Menu>
+      </Col>
+      <VerticalDivider />
+      <Space direction="vertical" size={40} style={{ flexGrow: 1 }}>
+        <Col span={20}>
+          {currentSettings === SettingsOptions.PROFILE && <ProfileSettings />}
+          {currentSettings === SettingsOptions.NOTIFICATIONS && (
```

Comment:
Would it be better to use url's routes (like `/settings/notifications` and `/settings/profit`) to determine which page to show so we could link directly to the correct one in the app. 

One example where this could be useful is when we want to get the "Do you want to enable notifications?" notification to link directly notification part of the settings page.

Reply:
you can link to it with a query param -- I didn't want to link to new pages cause it'd cause a reload of the page

Reply:
Gotcha. Linking it to a query param would be ideal if it's not too much work